extract_otp_secret_keys.py

```diff
@@ -46,18 +46,27 @@
 import fileinput
 import sys
 from urllib.parse import parse_qs, urlencode, urlparse, quote
-
+from os import path, mkdir
+from re import sub, compile as rcompile
 import generated_python.google_auth_pb2
 
 arg_parser = argparse.ArgumentParser()
 arg_parser.add_argument('--verbose', '-v', help='verbose output', action='store_true')
-arg_parser.add_argument('--qr', '-q', help='print QR codes (otpauth://...)', action='store_true')
+arg_parser.add_argument('--qr', '-q', help='generate QR codes (otpauth://...)', action='store_true')
```

Comment:
I suggest to remove --qr parameter and just use --saveqr and --printqr

Reply:
done